docs/google/floquet.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<p>for the sake of a simpler demo, can we hard-code filling_indices and remove as arguments from this and following functions?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
Done